docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,339 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
+* `interval` - how often to run the health check.
+    * This field is required.
+    * Example: `5s` (5 seconds)
+    * Example: `1m` (1 minute)
+    * Example: `250ms` (250 milliseconds)
+* `timout` - the connection timeout. Uses same format as interval.
+    * This field is required.
+    * Example: `10s` (10 seconds)
+* `actions` - how to react to health check result. Covered in more detail below.
+
+**Actions**
+
+Actions define how health checks results should be reacted to. Each check may have multiple actions. Actions support
+the following properties:
+
+* `trigger` - which kind of health check result to react to. Valid values include `pass`, `fail` and `change`.
+    * This field is required
+    * `change` is when the status changes from `pass` to `fail` or vice-versa.
+* `duration` - only trigger the action if the trigger state has existed for the given duration.
+    * This field is optional. If not specified, the duration is not checked.
+    * Example: `30s` (30 seconds)
+    * Use with `change` trigger events is not recommended.
+* `consecutiveEvents` - the number of consecutive results of the given trigger type before executing the action.
+    * This field is optional and defaults to 1
+    * Use with `change` trigger events is not recommended.
+* `action` - the action to take when the prerequisites defined by `trigger`, `duration` and `consecutiveEvents` are met.
+    * This field is required
+    * Valid actions include:
+        * `mark unhealthy` - sets the associated terminator's precedence to `failed`.
+        * `mark healthy` - sets the associated terminator's precedence back from `failed` to its original value.
+        * `increase cost N` - increases the cost of the associated terminator by `N`.
+        * `decrease cost N` - decreases the cost of the associated terminator by `N`.
+        * `send event` - causes a terminator event to be emitted from the controller. Useful for alerting or external integrations.
+
+**NOTE**
+
+Although multiple health checks can be configured, it's best if the actions don't overlap. If you have two health
+checks both changing the health status, the behavior when one check is passing and another is failing is undefined.
+It should generally be safe to have multiple checks adjusting cost or generating events.
+
+#### Active/Passive Fail-over
+
+We may also setups with primary and fail-over instances. These can be configured by setting the precedence in the
+config, rather than on the identity, as follows:
+
+```
+ziti edge update config test-host-config host.v2 --data '
+{
+    "terminators" : [
+        {
+            "address": "192.168.3.136",
+            "port" : 8080,
+            "protocol": "tcp",
+            "portChecks" : [ "health check definitions not shown for brevity" ],
+            "listenOptions" : {
+                "precedence" : "required"
+            }
+        },
+        {
+            "address": "192.168.3.137",
```

Comment:
is this IP intentionally not the same as the address shown above on line 395?

Reply:
Yes, it's showing pointing to two different application servers.